common/lib/symmath/symmath/symmath_check.py

```diff
@@ -198,10 +198,11 @@ def symmath_check(expect, ans, dynamath=None, options=None, debug=None, xml=None
             DEBUG = False
 
     # options
-    do_matrix = 'matrix' in (options or '')
-    do_qubit = 'qubit' in (options or '')
-    do_imaginary = 'imaginary' in (options or '')
```

Comment:
Yep. When I tried to fix the violation it was having, it failed another pylint check because the variable is unused. Can't find anything w/ that variable name in the whole platform.